lib/schemes/oauth2.js

```diff
@@ -64,16 +65,22 @@ export default class Oauth2Scheme {
   }
 
   login () {
```

Comment:
We can accept params with `login()`. So when we call it like  `loginWith('provider', { params: {} }`

Reply:
Right, that would be OK too. It's not currently supported as far as I can see (schemes/oauth2.js doesn't support arguments) but it would be easy to add. I'll drop this and create new PR.

Reply:
Args are always passed: https://github.com/nuxt-community/auth-module/blob/dev/lib/core/auth.js#L130 :)

Reply:
But are not handled by oath2 scheme: https://github.com/nuxt-community/auth-module/blob/dev/lib/schemes/oauth2.js#L66

Reply:
Yes. You can easily change signature to `login({ params }={})` :)